README.md

```diff
@@ -130,6 +130,16 @@ except NavaBaseError as e:
     print(str(e))
 ```
 
+### From Command Line Interface (CLI)
+```bash
+$ nava [--file FILE_PATH] [--loop] FILE_PATH
+```
+
+If you want to play sound asyncronously from CLI you can add `&`, like:
```

Comment:
This solution is not compatible with all operating systems! Also, I believe that async mode is meaningless in CLI, so I suggest removing this section from the README.md file.